webservices/tasks/utils.py

```diff
@@ -19,3 +22,13 @@ def get_bucket():
 
 def get_object(key):
     return get_bucket().Object(key=key)
+
+def get_s3_key(name):
+    connection = boto.s3.connect_to_region(
```

Comment:
Yes, this is summarized in the commit message for https://github.com/18F/openFEC/pull/2584/commits/028361c1b4b348d105c81dde6c9cba80b7e8b43e